service_management/azure/test/fixtures/management_certificate.pem

```diff
@@ -1,55 +0,0 @@
-Bag Attributes
```

Comment:
Thanks @vishrutshah. Looks like travis failed due to that, adding these files back.